### DISCUSSION TITLE
Is it possible to integrate Gauge with Espresso ?

### DISCUSSION THREAD

**Discussion by smaikk** [getgauge/gauge#1824] [General]
Is it possible to integrate Gauge framework with Espresso framework for android application?
I have not found any documentation or example which explains if possible to integrate the BDD of gauge with Espresso.

**Comment by zabil** [comment]
There are Gauge/Espresso examples yet. From Espresso examples it looks like is has some kind of android junit extension.

Technically Gauge can use any java library. You could try importing and using espresso as a library.

**Reply by smaikk** [reply]
Thank you for your quick response. Your positive response reassures me.

I have done all the combinations of the words "getgauge", "Gauge" and "Espresso",... on github and on google, but could not find any example that addresses this subject.

I will try to manually put the Gauge project folder in the "app" folder that appears at the Android root.
